src/common/components/ReviewModal/ReviewModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { Modal } from "@/common/components/Modal";
+import { type Review } from "@/common/components/ReviewItem";
+import { ProfileSchool } from "@/common/components/ReviewItem/ProfileSchool";
+import { ReviewBody } from "@/common/components/ReviewItem/ReviewBody";
+import ReviewHeader from "@/common/components/ReviewItem/ReviewHeader";
+import { reviewModalTheme } from "@/common/components/ReviewModal/ReviewModal.theme";
```

Comment:
chore(non-blocking): should use relative import for imports on the same level - storybook will sometimes complain about recursive import and break (see https://github.com/AfterClass-io/afterclass.io-v2/pull/98)

Reply:
thanks for pointing this out, fixed https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/79fb3d79570fd8a8455ef4e70dcc687af5bbf8b4